src/ceph-disk

```diff
@@ -2132,31 +2288,37 @@ def mount_activate(
     init,
     dmcrypt,
     dmcrypt_key_dir,
+    reactivate=False,
     ):
 
     if dmcrypt:
```

Comment:
could you please not re-indent this part of the code ? it will make it more difficult for backports